core/src/main/java/org/frankframework/http/rest/ApiListenerServlet.java

```diff
@@ -524,6 +522,11 @@ else if(segment.startsWith("{") && segment.endsWith("}")) {
 				 */
 				Message result = listener.processRequest(body, messageContext);
 
+				if (listener.getMethod().equals(ApiListener.HttpMethod.HEAD)) {
+					response.setContentType(listener.getContentType().toString());
+					result.isNull();
```

Comment:
What does `result.isNull()` here do?

Reply:
This is indeed wrong, do not really know why I did this, but I will change it